tsconfig.json

```diff
@@ -78,7 +78,7 @@
     /* Type Checking */
     "strict": true,                                      /* Enable all strict type-checking options. */
     "noImplicitAny": false,                            /* Enable error reporting for expressions and declarations with an implied 'any' type. */
-    // "strictNullChecks": true,                         /* When type checking, take into account 'null' and 'undefined'. */
+    "strictNullChecks": false,                         /* When type checking, take into account 'null' and 'undefined'. */
```

Comment:
Jakie jest powód żeby to wyłączyć? Wydaje się, że podpowiedzi kompilatora o tym, że coś może być null/undefined mogą być pomocne.

Reply:
Miałem problem z plikiem ts dlatego to wyłączyłem. Teraz już nie mam, więc włączam.